### PR TITLE
tsh logout + tctl create

### LINKED ISSUE #667: CLI polish
We've added quite a lot of new CLI commands in 2.0. Lets review and make sure that:

* `upsert` replaced with `create -f`
* Naming is consistent.
* `tsh logout` without arguments must allow you to log out of ALL proxy servers you're logged into

### PR DESCRIPTION
**Pull request by kontsevoy**
## Description

This commit closes #667 

- `tsh logout` will now log you out of everything
- `tctl upsert` has been renamed to `tctl create`

## TODO

`tctl create` does not have `--force` flag because implementing it for every resource type right now is not a high priority.

### CODE REVIEW

**Comment by kontsevoy** [comment]
retest please

**Comment by kontsevoy** [comment]
retest this please

File: lib/client/profile.go [hunk 1]
```diff
@@ -114,3 +115,40 @@ func (cp *ClientProfile) SaveTo(filePath string, opts ProfileOptions) error {
...
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	if len(keys) == 0 {
+		fmt.Printf("%s is not logged in\n", username)
+		return nil
+	}
+	// ... and delete them:
+	for _, key := range keys {
+		err = agent.DeleteKey(key.ProxyHost, username)
```
Comment (klizhentas) [inline lib/client/profile.go]: does it delete key from disk? I would assume the logout should delete the keys.

**Review by klizhentas — COMMENTED** [review]
looks good, have only one question about deleting the keys from disk.

File: lib/client/profile.go (hunk 1 above)
Comment (kontsevoy) [inline lib/client/profile.go]: also from SSH agent as well

**Comment by kontsevoy** [comment]
retest this please

**Pull request merged**